GameData/RealismOverhaul/RO_RecommendedMods/TACLS/RO_TACLS_Tanks.cfg

```diff
@@ -220,10 +377,37 @@
 	@description = A life support recycling module that uses the Sabatier reaction to reclaim water from carbon dioxide and hydrogen. Produces 2kg LqdMethane/day.
 	@MODULE[TacGenericConverter]
 	{
+		//conversionRate = 2.0	// Amount LqdMethane/Day
```

Comment:
@JPLRepo Am I correct in understanding that conversionRate is no longer used, no longer an option in TACLS?